src/lib/Zikula/Bundle/CoreBundle/EventListener/ThemeListener.php

```diff
@@ -95,23 +97,29 @@ public function setDefaultPageAssets(GetResponseEvent $event)
             return;
         }
         $basePath = $event->getRequest()->getBasePath();
+        $jquery = $this->env == 'prod' ? 'jquery.min.js' : 'jquery.js';
```

Comment:
I never know whether this should be determined by `env` or by the `debug` parameter.